### ISSUE TITLE
Yandex Browser ignores the translation ban

### ISSUE THREAD

**Issue by neznaika0** [sphinx-doc/sphinx#12420]
### Describe the bug

Based on https://github.com/sphinx-doc/sphinx/pull/4424 , https://github.com/sphinx-doc/sphinx/issues/5727

The Yandex Browser translator is still trying to translate blocks with the class `class="notranslate"`.
The code is translated, spaces are added, or an empty block is shown.

The solution is to add the `translate="no"` property.

### How to Reproduce

1. Open https://codeigniter4.github.io/userguide/testing/overview.html in YandexBrowser 24.4
2. Translate page `EN -> RU`

### Environment Information

```text
Platform:              linux; (Linux-6.7.12-amd64-x86_64-with-glibc2.38)
Python version:        3.11.9 (main, Apr 10 2024, 13:16:36) [GCC 13.2.0])
Python implementation: CPython
Sphinx version:        5.3.0
Docutils version:      0.19
Jinja2 version:        3.1.4
```

### Sphinx extensions

```python
extensions = [
	'sphinxcontrib.phpdomain',
	'sphinx_rtd_theme',
]
```

### Additional context

See screenshot bug.
![Screenshot Translated](https://github.com/sphinx-doc/sphinx/assets/17183530/c8672e5c-e272-41e3-9188-013185822b61)

**Comment by n-peugnet** [comment]
Indeed, it seems like it is now the standard way to request browsers not to translate part of a document: https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/translate
